main.py

```diff
@@ -0,0 +1,88 @@
+"""Only user inputs with predefined list of choices."""
+
+from event import *
+from manager import *
+from datetime import date
+
+
+class Main(object):
+
+    manager_instance = Manager()
+
+    while (1):
+        print "1: add_event  2: read_event  3: update_event  4: delete_event  5: upcoming_and_completed events  " \
+              "6: list_event_by_date_or_by_city  7: list_events_in_date_range  8: Exit"
+
+        choice = int(raw_input("choice:"))
+
+        if choice == 1:
+            event_instance = Event()
```

Comment:
Instead, you can create a constructor in the Event class which accepts all the attr values as parameters.